Tests/SystemTests/FileOperationsTest.swift

```diff
@@ -163,7 +163,7 @@ final class FileOperationsTest: XCTestCase {
 
 #if !os(Windows)
   func testResizeFile() throws {
-    let fd = try FileDescriptor.open("/tmp/\(UUID().uuidString).txt", .readWrite, options: [.create, .truncate], permissions: .ownerReadWrite)
+    let fd = try FileDescriptor.open(FilePath("/tmp/\(UUID().uuidString).txt"), .readWrite, options: [.create, .truncate], permissions: .ownerReadWrite)
```

Comment:
Hm. This feels like it indicates an API bug.

Reply:
The string-taking version should either be disfavored or updated to construct a file path.